static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -55,8 +55,15 @@ function Breadcrumbs() {
     useExtractedDomNodes({replay});
   const {data: frameToTrace, isFetching: isFetchingTraces} = useReplayPerfData({replay});
 
-  const startTimestampMs = replay?.getReplay()?.started_at?.getTime() ?? 0;
-  const frames = replay?.getChapterFrames();
+  const startTimestampMs =
+    replay?.getReplay()?.started_at?.getTime() ?? 0 + startTimeOffsetMs;
+  const frames = replay
+    ?.getChapterFrames()
+    .filter(
+      frame =>
+        frame.offsetMs >= startTimeOffsetMs &&
+        frame.offsetMs <= startTimeOffsetMs + durationMs
+    );
```

Comment:
Here's that re-filtering I was thinking would happen. getChapterFrames is memorized, and this list isn't now. I think the clipped knowledge needs to be inside the reader is we can.

Reply:
Good point - put this into into a useMemo.

I still agree that putting this into the reader is better. Once we get this feature complete enough to test and get feedback I'll circle back to my wip branch. If I can't get it working on the second round I'll reach out and see if you've got any ideas!